dtsync/message.go

```diff
@@ -57,15 +58,13 @@ func DecodeMessage(data []byte) (Message, error) {
 	for len(data) != 0 {
 		val, n, err := varint.FromUvarint(data)
 		if err != nil {
-			log.Errorf("Failed to decode message: cannot read number of multiadds: %s", err)
-			return Message{}, err
+			return Message{}, fmt.Errorf("cannot read number of multiadds: %s", err)
```

Comment:
prefer '%w' on Errorf of errors so that the error can be unwrapped to get the internal error. here and below

Reply:
Done